src/ATen/test/dlconvertor_test.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ATen/ATen.h"
+#include "ATen/dlconvertor.h"
+
+#include <iostream>
+#include <string.h>
+#include <sstream>
+#include "test_assert.h"
+
+using namespace at;
+
+static void test() {
+  {
+    std::cout << "dlconvertor: convert ATen to DLTensor" << std::endl;
+    Tensor a = CPU(at::kFloat).zeros({3,4});
```

Comment:
Probably want to test this by generating random data, convertin to/from dlpack and then checking equality.

Reply:
agreed!